File: Assignment_11/program2.c
/////////////////////////////////////////////////////////////////
//
// Name :          program2.c
// Discription :   Display pattern
// Input :         Integer
// Output :        Integer, Charecter
// Author :        Shivam Rajendra Kale
// Date :          20/05/2025
//
/////////////////////////////////////////////////////////////////

/*
    Start
        Accept number from user
            Display 5  #  4  #  3  #  2  #  1  #    
    Stop                  
*/

#include<stdio.h>

void Pattern(int iNo)
{
    int iCnt = 0;

    if(iNo < 0)
    {
        iNo = -iNo;
    }

    for(iCnt = iNo; iCnt >= 1; iCnt--)
    {
        printf("%d\t#\t",iCnt);
    }   
}

int main()
{
    int iValue = 0;

    printf("Enter number :");
    scanf("%d",&iValue);

    Pattern(iValue);   //function call

    return 0;
}

File: program2.java
/////////////////////////////////////////////////////////////////
//
//  File name :     program2.java
//  Descreption :   Input in java
//  Input :         Integer, String, float
//  Output :        Integer, String, float
//  Author :        Shivam Rajendra Kale
//  Date :          09/07/2025
//
//////////////////////////////////////////////////////////////////

/*
    Start
        Accept number from user
            & Display       
    Stop
*/

import java.util.*;

class program52
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter your name : ");
        String name = sobj.nextLine();

        System.out.println("Enter your age : ");
        int Age = sobj.nextInt();

        System.out.println("Enter your marks : ");
        float Marks = sobj.nextFloat();

        System.out.println("Name : "+name);
        System.out.println("Age : "+Age);
        System.out.println("Marks : "+Marks);
    }
}